docs/api-reference.md

```diff
@@ -100,7 +177,12 @@ By default, all bits and attributes (*for `Row` queries only*) are returned. In
 
 Creates a field in the given index with the given name.
 
-The request payload is in JSON, and may contain the `options` field. The `options` field is a JSON object which must contain a `type` along with the corresponding configuration options. 
+The request payload is in JSON, and may contain the `options` field. The `options` field is a JSON object which must contain a `type` and optionally the `keys`:
```

Comment:
remove "and optionally the `keys`"
perhaps instead we can include "(optional)" after the `keys` description.